examples/ola-recorder.cpp

```diff
@@ -57,6 +57,12 @@ DEFINE_uint32(duration, 0, "The length of time (seconds) to run for.");
 // 0 means infinite looping
 DEFINE_s_uint32(iterations, i, 1,
                 "The number of times to repeat the show, 0 means unlimited.");
+DEFINE_uint32(start, 0,
+              "Time (milliseconds) in show file to start playback from.");
+DEFINE_uint32(stop, 0,
+              "Time (milliseconds) in show file to stop playback at.  If"
```

Comment:
I think just one space at the end of the sentence, but I think I got caught out making this comment elsewhere, or that may be Python...

Reply:
My grade school typing teacher is sneering at me...

Checking elsewhere, the convention seems to be one space.